fatturae/settings.py

```diff
@@ -17,7 +17,7 @@
 SECRET_KEY = "t2r(bf!0(*4hq+2a4qg_=)l0zphz(da_+i=8dr@^_q&1pp(ut@"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env("DEBUG")
+DEBUG = True  # env("DEBUG")
```

Comment:
As I said for the `env.db` change (see below), can you revert back to `env("DEBUG")`? Basically this means "read the DEBUG variable from the `.env` file"

If you want to enable debug locally, in the `.env` file write:

`DEBUG=True`